ensembl_genes/Red.py

```diff
@@ -0,0 +1,264 @@
+""" Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
```

Comment:
I changed that because pylint complained about missing docstring for the module... but I'll change it back to # and then add an extra comment """ for the module under it. I'm not sure if pylint would complain again because it's expecting it to be in the first line. We'll see.